h/accounts/schemas.py

```diff
@@ -197,6 +199,20 @@ class SignupSchema(CSRFSchema):
     )
 
 
+class SignupSchema(CSRFSchema):
+    username = username_node()
+    email = email_node(title=_("Email address"))
+    password = new_password_node(title=_("Password"))
+    privacy_accepted = privacy_accepted_node()
+    comms_opt_in = comms_opt_in_node()
```

Comment:
`SignupSchema` is the same exact schema that it was before, just it now uses shared helper functions for all the nodes. This is the schema used for the normal, non-ORCID signup form.